### PR TITLE
Refs [TASK][34631] Create UI Weather Detail Screen

### PR DESCRIPTION
**Pull request by huydd-2678**
## Related Tickets
- [#Task 34631: Create UI Weather Detail Screen](https://edu-redmine.sun-asterisk.vn/issues/34631)
## WHAT
- Detail UI: https://marvelapp.com/prototype/6j24jfh/screen/77554060
- Create ImageView to show current weather icon
- Create TextViews to show weather informations
- Create TextView History
- Create RecyclerView to show weather history
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/item_history.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/textDate" />
+
+    <TextView
+        android:id="@+id/textDescription"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/title_description"
+        android:textSize="24sp"
```
Comment (KhacTung-Shjn) [inline app/src/main/res/layout/item_history.xml]: bỏ fix cứng đi nhé

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+    <string name="title_humidity">Humidity</string>
+    <string name="title_pressure">Pressure</string>
+    <string name="title_cloud">Cloud</string>
+    <string name="title_visibility">Visibility</string>
+    <string name="title_history">History</string>
+    <string name="title_description">Cloudy</string>
+    <string name="title_weather_detail">Weather Detail</string>
+
+    <string name="text_degree">°C</string>
+
```
Comment (KhacTung-Shjn) [inline app/src/main/res/values/strings.xml]: remove line đi

File: app/src/main/res/layout/item_history.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/dp_150"
```
Comment (datdt-1935) [inline app/src/main/res/layout/item_history.xml:5]: hạn chế fix cứng width-height nhé

**Review by datdt-1935 — COMMENTED** [review]
Format lại các file icon

**Review by tuannm-1276 — APPROVED** [review]

File: app/src/main/res/layout/item_history.xml (hunk 3 above)
Comment (huydd-2678) [reply app/src/main/res/layout/item_history.xml:5]: Chỗ này em không dùng cái khác được nên mới phải fix cứng ạ

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**